Makefile

```diff
@@ -145,9 +145,15 @@ push: checkvars
 artifacts: docker
 	@echo 'To be added'
 
-pilot/platform/kube/config:
+pilot/platform/kube/config: ${HOME}/.kube/config
 	ln -fs ~/.kube/config pilot/platform/kube/
 
+${HOME}/.kube/config: ${HOME}/.kube
+	touch $@
```

Comment:
Okay. this is bad compared to the other one. I prefer to modify stuff under istio/ folder, instead of touching anything in the user's home directory.